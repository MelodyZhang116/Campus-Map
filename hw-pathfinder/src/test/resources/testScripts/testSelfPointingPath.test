# test a path that start from a node and end with the same node.

CreateGraph g

AddNode g n1
AddNode g n2
AddNode g n3
AddEdge g n1 n2 135.798463321

FindPath g n1 n1
