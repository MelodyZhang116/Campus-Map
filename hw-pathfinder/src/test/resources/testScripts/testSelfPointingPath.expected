# test a path that start from a node and end with the same node.

created graph g

added node n1 to g
added node n2 to g
added node n3 to g
added edge 135.798 from n1 to n2 in g

path from n1 to n1:
total cost: 0.000